fastlane/lib/fastlane/actions/resign.rb

```diff
@@ -65,13 +65,23 @@ def self.available_options
                                        end),
           FastlaneCore::ConfigItem.new(key: :version,
                                        env_name: "FL_RESIGN_VERSION",
-                                       description: "Version number to force resigned ipa to use",
+                                       description: "Version number to force resigned ipa to use. Updates both CFBundleShortVersionString and CFBundleIdentifier values in Info.plist. Applies for main app and all nested apps or extensions",
                                        is_string: true,
                                        optional: true),
           FastlaneCore::ConfigItem.new(key: :display_name,
                                        env_name: "FL_DISPLAY_NAME",
                                        description: "Display name to force resigned ipa to use",
                                        is_string: true,
+                                       optional: true),
+          FastlaneCore::ConfigItem.new(key: :short_version,
```

Comment:
Use [`conflicting_options`](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/configuration/config_item.rb#L17) to ensure that `version` isn't used with `short_version` or `bundle_version`. I know this is redundant with the script but will be displayed more nicely for action users!